.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "cryptography"]
 	path = cryptography
 	url = git@github.com:PolymathNetwork/cryptography.git
-	branch = master 
+	branch = develop 
```

Comment:
I'm surprised the API changes in cryptography's `develop` didn't break this 🤔

Reply:
Yeah, we need to be diligent about committing and updating the submodule in this repo. Alternatively, we can change the CI options and instead of `git submodule update --init` use `git submodule update --init --remote` to fetch the latest version of the branch for the submodule.